### PR TITLE
Fixing owners name in services/cincinnati-ci/admin_rbac.yaml

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: services/cincinnati-ci/admin_rbac.yaml [hunk 1]
```diff
@@ -41,9 +41,8 @@ items:
     metadata:
       name: cincinnati-ci-admins
     users:
-      - steveeJ
       - steveej
-      - lalatendum
+      - lalatendumohanty
```
Comment (steveeJ) [inline services/cincinnati-ci/admin_rbac.yaml:45]:
```suggestion
      - LalatenduMohanty
```

File: services/cincinnati-ci/admin_rbac.yaml [hunk 2]
```diff
@@ -41,9 +41,8 @@ items:
     metadata:
       name: cincinnati-ci-admins
     users:
-      - steveeJ
       - steveej
```
Comment (steveeJ) [inline services/cincinnati-ci/admin_rbac.yaml:44]:
```suggestion
      - steveeJ
```

**Review by steveeJ — CHANGES_REQUESTED** [review]
I'd stick to your original GH username including the casing, and let prow handle the rest. I [fixed one bug pertaining to this](https://github.com/kubernetes/test-infra/pull/19062) a few months ago.

File: services/cincinnati-ci/admin_rbac.yaml (hunk 1 above)
Comment (LalatenduMohanty) [reply services/cincinnati-ci/admin_rbac.yaml]: Fixed in https://github.com/openshift/release/commit/79df66057e329c9ae2f10c7f8c1cef41e281f893

File: services/cincinnati-ci/admin_rbac.yaml (hunk 2 above)
Comment (LalatenduMohanty) [reply services/cincinnati-ci/admin_rbac.yaml]: Fixed in https://github.com/openshift/release/commit/79df66057e329c9ae2f10c7f8c1cef41e281f893

**Review by steveeJ — APPROVED** [review]
/lgtm

**Pull request merged**